### ISSUE TITLE
NBTApi + shading

### ISSUE THREAD

**Issue by ShaneBeee** [JorelAli/CommandAPI#174]
**CommandAPI version**
5.3

**Minecraft version**
1.16.4

**Describe the bug**
Hello, I just recently came across this API and decided to give it a try. Going very nicely, but I noticed if I use NBT-API and have it shaded in my plugin (as I like to do) it throws an error that its not recognized by this API

**My code**
This code reproduces the behavior:
(Instead here I am putting the error, as the code works if I manually add the NBT-API plugin to my server
```java
[23:52:37 WARN]: [CommandAPI] [CommandAPI] Couldn't hook into the NBTAPI for NBT support. Download it from https://www.spigotmc.org/resources/nbt-api.7939/
[23:52:37 INFO]: [CommandAPI] [CommandAPI] Hooked into Spigot successfully for Chat/ChatComponents
[23:52:37 ERROR]: [org.bukkit.craftbukkit.v1_16_R3.CraftServer] de/tr7zw/nbtapi/NBTCompound initializing TestPlugin v1.0-SNAPSHOT (Is it up to date?)
java.lang.NoClassDefFoundError: de/tr7zw/nbtapi/NBTCompound
	at dev.jorel.commandapi.arguments.NBTCompoundArgument.<init>(NBTCompoundArgument.java:19) ~[?:?]
	at com.shanebeestudios.test.TestPlugin.onLoad(TestPlugin.java:20) ~[?:?]
	at org.bukkit.craftbukkit.v1_16_R3.CraftServer.loadPlugins(CraftServer.java:394) ~[patched_1.16.4.jar:git-Paper-282]
	at net.minecraft.server.v1_16_R3.DedicatedServer.init(DedicatedServer.java:206) ~[patched_1.16.4.jar:git-Paper-282]
	at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:940) ~[patched_1.16.4.jar:git-Paper-282]
	at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:177) ~[patched_1.16.4.jar:git-Paper-282]
	at java.lang.Thread.run(Thread.java:832) [?:?]
Caused by: java.lang.ClassNotFoundException: de.tr7zw.nbtapi.NBTCompound
	at java.net.URLClassLoader.findClass(URLClassLoader.java:435) ~[?:?]
	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:171) ~[patched_1.16.4.jar:git-Paper-282]
	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:100) ~[patched_1.16.4.jar:git-Paper-282]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:589) ~[?:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:522) ~[?:?]
	... 7 more
```

**Expected behavior**
Im not sure if its possible to look for the API if it has been shaded and relocated.

**Screenshots**
none

**Additional context**
none

**Comment by JorelAli** [comment]
Hi, thanks for pointing this out. After close inspection, I've discovered that I've used the completely wrong version of the NBTAPI. Instead, the CommandAPI uses _item-nbt-api-plugin_ instead of _item-nbt-api!_ 

_(Whoops!)_

I'll see if I can fix this issue and if possible, push out a new release that fixes this ASAP.

**Comment by JorelAli** [comment]
Fixed in version 5.4.

**Comment by ShaneBeee** [comment]
Thank you very much, I really appreciate it 👍
